packages/docs/src/tags/interface.tsx

```diff
@@ -21,6 +21,9 @@ export class InterfaceTagRenderer {
     public render: TagRenderer = ({ value: name }, key, tagRenderers) => {
         const iface = this.propsStore.getInterface(name);
         const props = this.propsStore.getProps(iface);
+        if (iface === undefined) {
+            throw new Error(`Unknown @interface ${name}`);
```

Comment:
are you sure this should `throw` and not simply log a `console.error` and return null?